skpr/image-php
# Adds https://github.com/NoiseByNorthwest/php-spx

**Overview**

Adds SPX for debugging application performance issues for both command line and web requests.

https://github.com/NoiseByNorthwest/php-spx

**Testing**

1. Browse to http://127.0.0.1:8080/?SPX_UI_URI=/
2. Select "Enabled" and "Automatic start"
3. Browse your application
4. Check back at http://127.0.0.1:8080/?SPX_UI_URI=/
5. Click on a profile at the bottom of the page
